src/interpreter/bytecode.h

```diff
@@ -13,6 +13,7 @@
  * Bytecode has the following layout:
  *
  * Program header:
+ * - 32-bit verona magic number, 4085854915
```

Comment:
The number should be in hex, or doesn't even have to appear. Just add a "equal to the MAGIC_NUMBER constant" instead.